grid/chat/get-messages/profile.supr

```diff
@@ -14,10 +14,18 @@ usecase GetMessages {
 		destination! string!
 		
 		"""
-		AfterDate
-		Retrieve only messages after this date and time, UNIX timestamp format (e.g. 1095379198.75)
+		After Date
+		Retrieve only messages after this date and time, UNIX timestamp format (e.g. 1095379198.75) 
+		or snowflake format, which includes UNIX timestamp in its first 42 bits.
```

Comment:
Why do we need snowflake format here? It is provider specific. Do we return snowflake in the output to the client?

Reply:
`previousPage` and `nextPage` are snowflakes when used with discord, so yes it is provider specific. I did this to enable pagination forwards and backwards at least for discord.

Reply:
Since we can't properly check input types, I'll limit discord to forward pagination only and remove `string` type. @jnv are you okay with it?